Documentation/Programming-OpenPLC.rst

```diff
@@ -1,4 +1,4 @@
-Programming the Controller
+Programming the OpenPLC Board
 ==========================
 
 There are several programmers available to program any controller IC. They are called as ISPs. USBASP is one of them. Before programming the IC, we need to set the fuses. 
```

Comment:
"Before programming the IC, we need to set the fuses.": This does not sound proper.
suggested change be like:
"The micro-controller IC, we are using comprises of an internal calibrated RC oscillator. In order to use the external oscillator, one must set the fuses appropriately. This must be done only once if the IC is changed."